scripts/etc/init.d/rcS

```diff
@@ -760,7 +760,7 @@ cd /rootfs/boot/config/apt/
 # iterate through all the *.list files and add them to /etc/apt/sources.list.d
 for listfile in ./*.list
```

Comment:
if you feel so strongly about it, go ahead and merge... I already commented what I think is wrong with it, you provided some counter argument which I've shown to be wrong and provided you with code that will actually do what you seem to be trying to accomplish, namely not run the loop if no file matches the glob...